### ISSUE TITLE
Timing Diagrams

### ISSUE THREAD

**Issue by maxslug** [knsv/mermaid#615]
First off, THANK YOU!  This is a huge thing for documentation.  I'm very excited to find this library supported in Gitlab.

This issue is a feature request for timing diagrams.  If there was some way to merge https://github.com/wavedrom/wavedrom.github.io into here that would make us hardware engineers so, so very happy.

**Comment by gvlx** [comment]
I agree with @maxslug as those kinds of diagrams (timing and schematics) also interest me.

However, the usage of 'JSON' in 'wavedrom' is radically opposed to the simpler and more direct philosophy of this project.

Having said that, maybe I could think of a way to convert to/from both formats...

Keep me informed on this issue.

**Comment by Tnze** [comment]
I saw this before I was going to create an issue.🧐
